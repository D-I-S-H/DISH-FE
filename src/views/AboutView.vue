<template>
  <div class="container mt-5">
    <div class="card">
      <div class="card-body">
        <h5 class="card-title">About</h5>
        <p class="card-text">This is the about page</p>
      </div>
    </div>
  </div>
</template>
