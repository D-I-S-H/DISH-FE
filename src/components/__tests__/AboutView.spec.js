import { describe, it, expect } from "vitest";

import { mount } from "@vue/test-utils";
import AboutView from "../../views/AboutView.vue";

describe("Render about page", () => {
  it("renders properly", () => {
    const wrapper = mount(AboutView);
    expect(wrapper.html()).toMatchSnapshot();
  });
});
