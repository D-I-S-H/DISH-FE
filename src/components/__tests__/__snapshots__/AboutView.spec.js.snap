// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Render about page > renders properly 1`] = `
"<div class="container mt-5">
  <div class="card">
    <div class="card-body">
      <h5 class="card-title">About</h5>
      <p class="card-text"> Dining Information Served Here (D.I.S.H) provides simple and easy access to MTU's dining hall menus. </p>
    </div>
  </div>
</div>"
`;

exports[`Render test > renders properly 1`] = `
"<div class="container mt-5">
  <div class="card">
    <div class="card-body">
      <h5 class="card-title">About</h5>
      <p class="card-text"> Dining Information Served Here (D.I.S.H) provides simple and easy access to MTU's dining hall menus. </p>
    </div>
  </div>
</div>"
`;
